### ISSUE TITLE
When you get close - range functionality

### ISSUE THREAD

**Issue by mrquincle** [crownstone/CrownstoneApp#9]
Android.

The range functionality does have a few bugs.

When "Determining range.." it displays the following message, even if I'm really close.

```
I'm not sure yet... 
I could not collect enough points.. Could you try again?
```

If I then go back it continues with "Determining range" even on top of the main window.

**Comment by AlexDM0** [comment]
This is the result of a mismatch in background handling between android and ios. We will address this asap. Thanks!

**Comment by vliedel** [comment]
Should be fixed in version 0.7.3
